### DISCUSSION TITLE
Reload the same page

### DISCUSSION THREAD

**Discussion by Afnan998782** [vercel/community#2017] [Help]
### Summary

Hey. I hope you are doing well 
the promblem is when i click on the next page like it loads up the same page again and again
Here is my vercel.json code 
{
  "builds": [
    { "src": "index.html", "use": "@vercel/static" },
    { "src": "index2.html", "use": "@vercel/static" }
  ],
  "routes": [
    { "src": "/index2", "dest": "/index2.html" },
    { "src": "/(.*)", "dest": "/index.html" }
  ]
}

**Comment by amyegan** [comment]
Hi @Afnan998782. I wasn't able to reproduce this problem in my basic static test app. Can you share an example deployment link or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to help me get a better look at the problem?
